mmdet3d/models/backbones/dgcnn.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.runner import BaseModule, auto_fp16
+from torch import nn as nn
+
+from mmdet3d.ops import DGCNNFAModule, DGCNNGFModule
+from mmdet.models import BACKBONES
+
+
+@BACKBONES.register_module()
+class DGCNN(BaseModule):
```

Comment:
DGCNN -> DGCNNBackbone, thus we could distinguish it from the method/model of DGCNN